api/models/organization.js

```diff
@@ -85,6 +86,20 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false,
       defaultValue: false,
     },
+    sandboxNextCleaningAt: {
```

Comment:
We may want to consider validating that this column is NOT null if the org is a sandbox (https://sequelize.org/master/manual/validations-and-constraints.html#model-wide-validations)

This also means that if the value of `isSandbox` changes we need to make sure to update this value accordingly.

We could look at implementing this and providing default values with a `beforeValidate` "hook" (https://sequelize.org/master/manual/hooks.html). Ex.
```
  // in the Organization model
  ...
  beforeValidate: (org, options) => {
    if (!org.isSandbox) {
      org.sandboxNextCleaningAt = null;
    } else if (!org.sandboxNextCleaningAt) {
        org.sandboxNextCleaningAt = moment().add(sandboxDays, 'days').endOf('day');
    }
  },
  ...
```

Then we also don't need to handle this explicitly when creating an Org.

Reply:
awesome suggestion ...  thanks! 👍🏿